libr/socket/r2pipe.c

```diff
@@ -304,7 +304,6 @@ R_API R2Pipe *r2pipe_open(const char *cmd) {
 			fflush (stderr);
 			// trigger the blocking read
 			write (1, "\xff", 1);
-			close (r2p->output[0]);
```

Comment:
It's created only if `#if __UNIX__` is defined in https://github.com/radareorg/radare2/pull/17617/files#diff-29c03378d6a5a5012e9a4b4ba0daaadaR201
What about wrapping this in `#if !__UNIX__` instead?

Reply:
Mmm, no r2p->output[0] is set to -1 just few lines above, at line 301, so I think it is ok here.